.github/workflows/main.yml

```diff
@@ -62,7 +62,10 @@ jobs:
           mkdir -p "${GITHUB_WORKSPACE}/coverage"
           pushd build/src/CMakeFiles/hermes.dir
           lcov -c -d . -o "${GITHUB_WORKSPACE}/coverage/lcov.info"
-
+          popd
+          mkdir -p "${GITHUB_WORKSPACE}/stdio_coverage"
+          pushd build/adapter/src/hermes/adapter/stdio/CMakeFiles/hermes_stdio.dir
+          lcov -c -d . -o "${GITHUB_WORKSPACE}/stdio_coverage/lcov.info"
```

Comment:
The coveralls action we use looks for a file called `./coverage/lcov.info` by default, so the stdio coverage report you generate gets ignored. `lcov` has an [--add-tracefile](https://linux.die.net/man/1/lcov) option you can use to combine the multiple coverage files into one.

Reply:
So the command should be 

```
lcov --add-tracefile -c -d . -o "${GITHUB_WORKSPACE}/stdio_coverage/lcov.info"
```

Reply:
I'm not sure off the top of my head, but the idea is to create two coverage files (one for the `src` directory and one for the `adapter` directory) and then add them together with the `--add-tracefile` feature leaving a single file at `./coverage/lcov.info`. You'll have to read the man page I linked above to figure it out. Coverage reports for your fork are [here](https://coveralls.io/github/hariharan-devarajan/hermes), so just check there and make sure that the command you use lists both `src` and `adapter` in the tree view. Coverage reports for this PR show up at the bottom of [this page](https://coveralls.io/github/HDFGroup/hermes), though I'm not sure if it generates a new report for each commit.

Reply:
Added.

Reply:
I have tested the change and it worked. Just look at `main.yml`
https://github.com/HDFGroup/hermes/compare/master...kimmy/coverall

Reply:
I have updated the PR based on kimmy's comment.